### PR TITLE
DOC: Updated Resources section with Other Rubrics

### PR DESCRIPTION
**Pull request by lacabra**
Title says it all.

### CODE REVIEW

File: Resources/README.md [hunk 1]
```diff
@@ -12,3 +12,12 @@ information and data that DIAL has collected about the various categories and in
 that may be collected, including how these indicators map to existing maturity models,
 how they may be collected, and their relevance.
 
+## Others Rubrics
```
Comment (conradsp) [inline Resources/README.md]: Should read 'Other Rubrics'

File: Resources/README.md [hunk 2]
```diff
@@ -12,3 +12,12 @@ information and data that DIAL has collected about the various categories and in
 that may be collected, including how these indicators map to existing maturity models,
 how they may be collected, and their relevance.
 
+## Others Rubrics 
+
+Other rubrics and evaluation frameworks that have been reviewed in the process of consolidating this maturity rubric are listed below:
+
+* [Digital Health Software: Global Good Maturity Model](https://wiki.digitalsquare.io/index.php/What_are_Global_Goods) developed by Digital Square.
+* [Maturity Model](https://www.osc.dial.community/maturity-model.html) developed by DIAL's Open Source Center based of the [Apache Project Maturity Model](https://community.apache.org/apache-way/apache-project-maturity-model.html)
```
Comment (conradsp) [inline Resources/README.md]: 'based on'

**Review by conradsp — APPROVED** [review]

File: Resources/README.md (hunk 1 above)
Comment (lacabra) [reply Resources/README.md]: Thanks, fixed in af296c3

File: Resources/README.md (hunk 2 above)
Comment (lacabra) [reply Resources/README.md]: Thanks, fixed in af296c3

**Pull request merged**